python/grass/gunittest/main.py

```diff
@@ -224,6 +224,7 @@ def main():
     invoker = GrassTestFilesInvoker(
         start_dir=start_dir,
         file_anonymizer=FileAnonymizer(paths_to_remove=[abs_start_dir]),
+        timeout=config.getfloat("timeout", None),
```

Comment:
This seems to cause troubles, see

https://github.com/OSGeo/grass-addons/runs/5097252329?check_suite_focus=true

```
 Executing <python3 -m grass.gunittest.main --grassdata /home/runner --location nc_spm_full_v2alpha2 --location-type nc --min-success 60> ...
Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/home/runner/install/grass81/etc/python/grass/gunittest/main.py", line 248, in <module>
    sys.exit(main())
  File "/home/runner/install/grass81/etc/python/grass/gunittest/main.py", line 227, in main
    timeout=config.getfloat("timeout", None),
AttributeError: 'dict' object has no attribute 'getfloat'
Execution of <python3 -m grass.gunittest.main --grassdata /home/runner --location nc_spm_full_v2alpha2 --location-type nc --min-success 60> finished.
Cleaning up temporary files...
Error: Process completed with exit code 1.
```

Reply:
Thanks for reporting this. I'll look into that and submit a fix. Python 3.8 is what is used here, but `'dict' object has no attribute 'getfloat'` is pretty clear. Probably happens when the config file is not there.

Reply:
Fix should be in #2180.